_posts/2020-01-07-jump_dev_2020.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "JuMP-dev 2020: Abstract Submission"
+date:   2020-01-07
+categories: [jump-dev]
+---
+
+June 15-17, 2019, [Louvain-la-Neuve, Belgium](https://goo.gl/maps/fyxqevrKn76nvhwq9).
+
+This workshop is the fourth JuMP-dev workshop after the [first](/meetings/mit2017/),
+[second](/meetings/bordeaux2018/), and [third](/meetings/santiago2019) JuMP-dev workshops.
+
+The purpose of the workshop is to bring together students, researchers, and practitioners
+with interests in the software aspects of [JuMP](https://github.com/JuliaOpt/JuMP.jl) and
+related packages. The workshop is open for anybody to attend. In particular, we invite new
+contributors and those who have not met the core development team.
+
+## Abstract Submission
+
+**[Submit an abstract.](https://docs.google.com/forms/d/e/1FAIpQLSd7d3TJ2de0GSD56lEBrR8e6KmVlPANjwNsr09KroSAuIq-Xw/viewform?usp=sf_link)**
```

Comment:
Under "I want to give a", perhaps note that the committee might decide to give a slot for a short talk even if you asked for a standard talk. This could also be worded as "Preferred talk length"